### PR TITLE
Support symfony 7

### PR DESCRIPTION
**Pull request by Tom32i**
Support Symfony 7

### CODE REVIEW

**Review by ogizanagi — APPROVED** [review]
Awesome work, thank you 👏🏻 

I've pushed some additional things as well

File: src/Service/Git/LastModifiedFetcher.php [hunk 1]
```diff
@@ -45,7 +46,18 @@ public function __invoke(string $filePath): ?\DateTimeImmutable
         if (null === self::$gitAvailable) {
             // Check once if the git command is available
             $process = new Process([...$executable, '--version']);
-            $process->run();
+
+            try {
+                $process->run();
+            } catch (ProcessFailedException $exception) {
```
[the remark this answers is not in the document]
Comment (Tom32i) [reply src/Service/Git/LastModifiedFetcher.php:52]: That was me trying to fix compatibility with Sylfony 7.1: https://github.com/StenopePHP/Stenope/pull/186/commits/d61c7421b904b73592ff6fcf39ebc0ccd01cda59

Maybe this is not necessary anymore?

[the remark this answers is not in the document]
Comment (ogizanagi) [reply src/Service/Git/LastModifiedFetcher.php:52]: Oh, I see:

```shell
 make install.71
make test
symfony php vendor/bin/simple-phpunit
PHPUnit 9.6.18 by Sebastian Bergmann and contributors.

Testing
...............................................................  63 / 150 ( 42%)
..............................SSSSS............................ 126 / 150 ( 84%)
.E....S.S...............                                        150 / 150 (100%)

Time: 00:02.053, Memory: 46.50 MB

There was 1 error:

1) Stenope\Bundle\Tests\Unit\Service\Git\LastModifiedFetcherTest::testUnavailable
Symfony\Component\Process\Exception\ProcessStartFailedException: The command "'not-valid-path' '--version'" failed.

Working directory: /Users/ogi/Workspace/OSS/StenopePHP/stenope

Error: proc_open(): posix_spawn() failed: No such file or directory

/Users/ogi/Workspace/OSS/StenopePHP/stenope/vendor/symfony/process/Process.php:356
/Users/ogi/Workspace/OSS/StenopePHP/stenope/vendor/symfony/process/Process.php:248
/Users/ogi/Workspace/OSS/StenopePHP/stenope/src/Service/Git/LastModifiedFetcher.php:50
/Users/ogi/Workspace/OSS/StenopePHP/stenope/tests/Unit/Service/Git/LastModifiedFetcherTest.php:43
/Users/ogi/Workspace/OSS/StenopePHP/stenope/vendor/phpunit/phpunit/src/Framework/TestResult.php:729
/Users/ogi/Workspace/OSS/StenopePHP/stenope/vendor/phpunit/phpunit/src/Framework/TestSuite.php:685
/Users/ogi/Workspace/OSS/StenopePHP/stenope/vendor/phpunit/phpunit/src/Framework/TestSuite.php:685
/Users/ogi/Workspace/OSS/StenopePHP/stenope/vendor/phpunit/phpunit/src/Framework/TestSuite.php:685
/Users/ogi/Workspace/OSS/StenopePHP/stenope/vendor/phpunit/phpunit/src/TextUI/TestRunner.php:651
```

I'll suggest the upper, different warning about this.

**Comment by ogizanagi** [comment]
Thanks @Tom32i.

**Pull request merged**